apps/rod/views/reuse-main-applicant-address.html

```diff
@@ -0,0 +1,21 @@
+{{<partials-page}}
+  {{$page-content}}
+  {{#markdown}}delivery-address-for-documents{{/markdown}} 
+
+  <div class="govuk-details__text">
+  <div class="govuk-summary-list__row summary-list-box" >
+        <div>{{values.main-applicant-address-1}}</div>
+        <div>{{values.main-applicant-address-2}}</div>
+        <div>{{values.main-applicant-town-or-city}}</div>
+        <div>{{values.main-applicant-postcode}}</div>
+    </div>
+  </div>
+  <br>
```

Comment:
Instead of a <br> tag, we can look to apply some margin on radios when they are used with `is-passport-return-address-group` id.